File: lua/1013.lua
local a = io.read("*n")
local b = io.read("*n")
local c = io.read("*n")

local m = ( a + b + math.abs(a-b) ) / 2
m = ( m + c + math.abs(m-c) ) / 2

m = string.format("%.0f", m)

print(m.." eh o maior")

File: go/1013.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c, m float64
	fmt.Scanf("%f", &a)
	fmt.Scanf("%f", &b)
	fmt.Scanf("%f", &c)
	m = (a + b + math.Abs(a-b)) / 2
	m = (m + c + math.Abs(m-c)) / 2
	fmt.Printf("%.0f eh o maior\n", m)
}
